classes/trait/driver/field/choices/single.php

```diff
@@ -37,13 +36,7 @@ public function renderExportValue(Model_Answer_Field $answerField)
         // Gets the answer value
         $selectedValue = $this->sanitizeValue($answerField->value);
 
-        // Gets the choices list
-        $choices = $this->getChoicesList();
-
-        $export    = '';
-        if( array_key_exists($selectedValue, $choices) ){
-            $export = e($choices[$selectedValue]);
-        }
+        $export = $this->getValueChoiceLabel($selectedValue);
 
         return $export;
```

Comment:
It's a value for a CSV file and it's already escaped outside the driver (and not with `e()` which HTML escapes).